### PR TITLE
Feature/create db schemas

### PR DESCRIPTION
**Pull request by robforsythe**
I included fields that we don't currently need. I'm not sure whether or not we want to have the bare minimum number of fields right now and if we choose to extend functionality, we add more later OR just add support for the fields now

### CODE REVIEW

File: db_schemas/user.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		"required": false
+	},
+	"email": {
+		"description": "The user's email address",
+		"type": "string",
+		"comment": "",
+		"required": false
+	},
+	"location": {
+		"description": "The ID of the user's location. Foreign Key (Location table).",
```
Comment (warrenharper) [inline db_schemas/user.json]: Which field in the location table is this referring to? @robforsythe

Comment (warrenharper) [inline db_schemas/user.json]: Should this be an integer

**Comment by warrenharper** [comment]
@robforsythe I like that idea of adding support for the extra fields now.

File: db_schemas/user.json (hunk 1 above)
Comment (robforsythe) [inline db_schemas/user.json]: it's an integer since we're using an ID as a primary key in the repository table.

Comment (robforsythe) [inline db_schemas/user.json]: yup! good catch.

**Comment by warrenharper** [comment]
Looks good to me

**Pull request merged**